test cases/frameworks/15 llvm/test.json

```diff
@@ -2,9 +2,8 @@
   "matrix": {
     "options": {
       "method": [
-        { "val": "config-tool", "skip_on_jobname": ["msys2-gcc"]},
-        { "val": "cmake", "skip_on_jobname": ["msys2-gcc"] },
-        { "val": "combination", "skip_on_jobname": ["msys2-gcc"]}
+        { "val": "cmake", "skip_on_jobname": ["msys2-clangx64ninja"] },
+        { "val": "combination", "skip_on_jobname": ["msys2-clangx64ninja"]}
```

Comment:
Are we totally dropping config-tool *everywhere*?

Reply:
Ah, sorry, typo.